<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE concept PUBLIC "-//OASIS//DTD DITA Concept//EN" "concept.dtd">
<concept id="concept_rp1_ghd_zs">
 <title>HBase</title>
 <conbody>
  <p>
   <dl>
    <dlentry>
     <dt>I get the following error when validating or starting a pipeline with an HBase
      destination:</dt>
     <dd>
      <codeblock>HBASE_06 - Cannot connect to cluster: org.apache.hadoop.hbase.MasterNotRunningException: 
com.google.protobuf.ServiceException: org.apache.hadoop.hbase.exceptions.ConnectionClosingException: 
Call to node00.local/&lt;IP_address>:60000 failed on local exception: 
org.apache.hadoop.hbase.exceptions.ConnectionClosingException: 
Connection to node00.local/&lt;IP_address>:60000 is closing. Call id=0, waitTime=58</codeblock>
     </dd>
     <dd>Is your HBase master is running? If so, then you might trying to connect to a secure HBase
      cluster without configuring the HBase destination to use Kerberos authentication. In the HBase
      destination properties, select <uicontrol>Kerberos Authentication</uicontrol> and try
      again.</dd>
    </dlentry>
   </dl>
  </p>
 </conbody>
</concept>
